### DISCUSSION TITLE
MAC Filter Configuration - Whitelist MAC Addresses

### DISCUSSION THREAD

**Discussion by nehasjains** [AdguardTeam/AdGuardHome#5455] [Q&A]
I have installed AdGuardHome on my raspberry pi 4. I disabled DHCP on my router and enabled it on AdGuardHome.

I have added MAC address and ip for devices under "DHCP static leases" settings. 

Now please guide me how can I whitelist MAC addresses so that no other device can connect to the router and use wifi.

Kindly guide.
Regards
Neha

**Comment by ishanjain28** [comment]
@nehasjains AGH is not a firewall and you should not be using it for that purpose. Most new devices come with mac randomization and have a toggle to use a random mac address every time it connects. This will easily bypass your efforts here. So for a start, Improve security of your network elsewhere to stop this from happening.

**Comment by Stanojoski** [comment]
MAC filtering is actually a good option. I need it at the moment. I have a tv box that should not get IP from the DHCP, otherwise, it won't work. 
whitelist/blacklist option is actually a good idea
